Dockerfile

```diff
@@ -15,7 +15,14 @@ WORKDIR /app
 COPY docker/requirements.txt /app/docker/
 COPY requirements.txt /app/
 
-RUN pip install --disable-pip-version-check --no-cache-dir --requirement requirements.txt --requirement docker/requirements.txt
+RUN apt update && \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/app/pyramid_oereb/pyramid_oereb/pullRequest?prid=4515276)